spec/swagger_helper.rb

```diff
@@ -64,11 +64,11 @@
               job_title: { type: :string, example: "Teacher of Geography" },
               skills_and_experience: { type: :string, example: "We're looking for a dedicated Teacher of Geography" },
               salary: { type: :string, example: "£12,345 to £67, 890" },
-              benefits_details: { type: :string, example: "TLR2a" },
-              starts_on: { type: :string, example: "Summer Term" },
-              external_reference: { type: :string, example: "123GTZY" },
-              visa_sponsorship_available: { type: :boolean },
-              is_job_share: { type: :boolean },
+              benefits_details: { type: :string, example: "TLR2a", nullable: true },
+              starts_on: { type: :string, example: "Summer Term", nullable: true },
+              external_reference: { type: :string, example: "123GTZY", nullable: true },
+              visa_sponsorship_available: { type: :boolean, nullable: true },
```

Comment:
IMHO these shouldn't be nullable - they are just optional properties. Otherwise we have 2 'null' states - missing and present null.